src/components/stepFour/index.js

```diff
@@ -314,6 +314,7 @@ const { PUBLISH } = NAVIGATION_STEPS
       let newState = { ...this.state }
       newState.loading = true;
       this.setState(newState);
+      contractStore.crowdsale.networkID = _networkID;
```

Comment:
Changing state in mobx works by using the methods in the contract store to update the properties.

setContractProperty('crowdsale', 'networkID', _networkID) would be the method signature in this case.